src/core/main.c

```diff
@@ -70,22 +70,25 @@ int main(int argc, char *argv[]) {
 	ret = sd_bus_open_user(&bus);
 	if (ret < 0) {
 		logprint(ERROR, "dbus: failed to connect to user bus: %s", strerror(-ret));
-		goto error;
+		exit(EXIT_FAILURE);
```

Comment:
We can just `return EXIT_FAILURE` instead of calling `exit`

Reply:
Done